### PR TITLE
Fixed the spacing of the header for the diff page sizes

### PR DESCRIPTION
**Pull request by TurnbulA**
Changed the spacing on the header made it look closer to the design
Made it act responsively 
Accidentally made the rest of the page more responsive

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -60,30 +64,33 @@ body {
...
 	position: absolute;
-	width: 10rem;
-	height: 10rem;
+	width: 9rem;
+	height: 9rem;
 	padding: 3rem auto 0 auto;
 	border-radius: 50%;
-	margin-left: -40rem;
-	margin-top: 4rem;
+	margin-left: -25rem;
```
Comment (mali891) [inline style.css]: What does`-25rem` do here?

File: style.css [hunk 2]
```diff
@@ -60,30 +64,33 @@ body {
...
 	display: flex;
 	justify-content: center;
 	border: solid;
 	height: 5rem;
 	width: 5rem;
-	margin-top: 2.5rem;
-	padding:0 !important;
+	margin-top: 2rem;
+	margin-left: 2rem;
+	padding: 0 !important;
```
Comment (mali891) [inline style.css]: Do you know why `!important` is required here?

File: style.css [hunk 3]
```diff
@@ -337,4 +345,39 @@ option{
 		text-align: left;
 		padding-left: 2rem;		
 	}
+	.row{
+		display: flex;
+		justify-content: center;
+		flex-direction: column;
+		width:114%;
```
Comment (mali891) [inline style.css]: What does `width: 114%` do here?

File: style.css [hunk 4]
```diff
@@ -337,4 +345,39 @@ option{
...
+	}
+	.section-head{
+		height: 70rem;
+	}
+	.button-border{
+		margin-top: 8rem;
+		margin-left: -20rem;
```
Comment (mali891) [inline style.css:379]: What does`-20rem` do here?

File: style.css (hunk 1 above)
Comment (TurnbulA) [reply style.css]: Positions the play button correctly, in the top left corner

File: style.css (hunk 2 above)
Comment (TurnbulA) [reply style.css]: It corrects the shape of the triangle in the play button, without it there is padding in the square. I don't know where that padding came from.

File: style.css (hunk 4 above)
Comment (TurnbulA) [reply style.css:379]: Positions play  button to top left when screen is below 992px

File: style.css (hunk 1 above)
Comment (mali891) [reply style.css]: Okay let's catch up on Monday about this because there are better ways of positioning elements

File: style.css (hunk 4 above)
Comment (mali891) [reply style.css:379]: Okay let's catch up on Monday about this because there are better ways of positioning elements

File: style.css [hunk 5]
```diff
@@ -345,16 +348,24 @@ option{
...
+	}
+	.header-button{
+		margin: 0 auto;
+	}
 	.row{
 		display: flex;
 		justify-content: center;
 		flex-direction: column;
-		width:114%;
+		width:100;
```
Comment (mali891) [inline style.css]: What does `width: 100` do?

File: style.css [hunk 6]
```diff
@@ -345,16 +348,24 @@ option{
 		text-align: left;
 		padding-left: 2rem;		
 	}
+	.header{
+		width:100%;
```
Comment (mali891) [inline style.css]: Space after colon or no space?

File: style.css [hunk 7]
```diff
@@ -345,16 +348,24 @@ option{
...
+	}
+	.header-button{
+		margin: 0 auto;
```
Comment (mali891) [inline style.css:357]: Space after colon or no space?

File: style.css [hunk 8]
```diff
@@ -345,16 +348,24 @@ option{
...
 		flex-direction: column;
-		width:114%;
+		width:100;
 	}
 	.section-head{
 		height: 70rem;
 	}
 	.button-border{
+		display: flex;
+		justify-content:left;
```
Comment (mali891) [inline style.css]: Space after colon or no space?

File: style.css (hunk 2 above)
Comment (mali891) [reply style.css]: Please resolve this so you don't need to use `!important`

File: style.css (hunk 3 above)
Comment (mali891) [reply style.css]: ?

File: style.css (hunk 5 above)
Comment (TurnbulA) [reply style.css]: Supposed to be 100% have fixed

File: style.css (hunk 6 above)
Comment (TurnbulA) [reply style.css]: Gone through and added spacing

**Review by mali891 — APPROVED** [review]

**Pull request merged**